blocks/avm-report/avm-report.css

```diff
@@ -71,4 +97,13 @@
     flex: unset;
     margin: 0;
   }
+
+  .avm-report-container .cards-wrapper {
```

Comment:
one block should not change the CSS of another

Reply:
Have moved these styles into the cards block